### ISSUE TITLE
Emacspeak initialization error on Mac OS due to Foundation package 

### ISSUE THREAD

**Issue by Newton-Climate** [tvraman/emacspeak#74]
Hi there, I was trying to install Emacspeak on Mac OS on my Apple M1 with OS 12.5. When I initialize emacspeak in the terminal, I hear the "Welcome to Emacspeak" voice, but VoiceOver doesn't work with the program. I went to emacspeak/server/Mac and got an error that said "package Foundation not found." I think it's related to the Xcode and PyObjc. I used pip to install PyObjC, which should include the Foundation module, but it seems to still not work, alongside looking for the Foundation module directly. PyObjC is importing via python but Foundation is not. Does anyone know how to get this module working? Thanks in advance 

Best,

Newton

**Comment by robertmeta** [comment]
Getting it working on MacOS 12+ was a bit of a slog for me as well, I wrote a guide: https://gist.github.com/robertmeta/6c370860c023f63350ba5e59a4a2a6d1

**Comment by Newton-Climate** [comment]
Hi Robert, I Have a quick question for you. Which python version do you have installed and running emacspeak? I tried python 3.9+, and it doesn’t work. The way I solved this problem with the Foundation library and PyObjC was by using Python 3.7 in a separate anaconda environment.

thanks for sending along that link and taking the time to write it.

Best,

Newton

**Comment by robertmeta** [comment]
Mine is 3.10.8

**Comment by robertmeta** [comment]
@Newton-Climate just curious if you got up and running?

**Comment by Newton-Climate** [comment]
Hi Robert, thanks for following up. I didn’t get it working on python 3.9+ after following the instructions unfortunately. I was still getting the same error about the Foundation package. Maybe it’s related to my Apple M1 chip or the new OS X?

I just made a new Conda environment with Python 3.7 to make it work.

Best,

Newton

**Comment by robertmeta** [comment]
Do you happen to use zsh, or any alternative shell?  I added instructions for dealing with an oddity where emacs will work when run from the terminal, but not via the built-in finder runner in OSX.  

- pip3 install pyobjc 

vs. 

- env -i pip3 install pyobjc

**Comment by robertmeta** [comment]
@tvraman I think this issue can be closed, the original user got it working, and we have updated the python shebang a couple times since then.  This issue likely has to do with lots of fiddly bits regarding multiple python versions on mac.  I think my comment above will fix 9/10 of these cases (install the pip3 libraries with and without env setup).
